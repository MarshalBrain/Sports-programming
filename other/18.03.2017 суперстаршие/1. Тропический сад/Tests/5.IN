3
1.5 -0.658
-2 3.828
-0.6 -1.973