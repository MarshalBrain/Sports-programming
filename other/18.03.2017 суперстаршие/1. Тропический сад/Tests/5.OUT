3.000
-1.000 1.000